kinetic/distribution.yaml

```diff
@@ -8339,6 +8339,24 @@ repositories:
       type: git
       url: https://bitbucket.org/osrf/gear.git
       version: master
+  ouster-release:
```

Comment:
This is kind of a strange name for the top-level repository, especially since it seems to conflate the source repository with the release one.  It wouldn't actually matter for the package names (since they would end up being `ros-kinetic-ouster-driver`), but for our future reviewing sanity I'm going to request that this just be `ouster`.

Reply:
Thanks for the review. Updated